src/dune_rules/lib.ml

```diff
@@ -1905,10 +1906,6 @@ let to_dune_lib ({ info; _ } as lib) ~modules ~foreign_objects ~dir :
     | None -> assert false
     | Some obj_dir -> Obj_dir.convert_to_external ~dir obj_dir
   in
-  let modules =
```

Comment:
moved to Lib_info, L519 (https://github.com/ocaml/dune/pull/6605/files#diff-3e8f844f93ba88bad4253c28d71937743695a26bd79de0378fbcafd2461186ebR519)